<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSC_SolutionChallenge_EReceipt_Device</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Item_List">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>211</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>T E A V A N A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_vanilla">
        <property name="text">
         <string>Vanilla Black Tea Latte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_green">
        <property name="text">
         <string>Green Tea Latte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_lemon">
        <property name="text">
         <string>Lemon Mint Tea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_lime">
        <property name="text">
         <string>Lime Passion Tea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_grapefruite">
        <property name="text">
         <string>Grapefruit Honey Black Tea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_chai">
        <property name="text">
         <string>Chai Tea Latte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tea_earl">
        <property name="text">
         <string>Earl Grey Tea Latte</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Frappuccino</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_java">
        <property name="text">
         <string>Java Chip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_white">
        <property name="text">
         <string>White Chocolate Mocha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_mocha">
        <property name="text">
         <string>Mocha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_green">
        <property name="text">
         <string>Green Tea Cream</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_chocolate">
        <property name="text">
         <string>Chocolate Cream Chip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_strawberries">
        <property name="text">
         <string>Strawberries &amp; Cream</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fra_avocado">
        <property name="text">
         <string>Avocado</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fizzio</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fiz_pink">
        <property name="text">
         <string>Pink Grapefruite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fiz_lime">
        <property name="text">
         <string>Cool Lime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fiz_black">
        <property name="text">
         <string>Black Tea Lemonade</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Chocolate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cho_signature">
        <property name="text">
         <string>Signature Chocolate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>211</width>
       <height>293</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Coffee &amp; Espresso</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_cappuccino">
        <property name="text">
         <string>Cold Foam Cappuccino</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_caramel">
        <property name="text">
         <string>Caramel Macchiato</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_mocha">
        <property name="text">
         <string>Caffe Mocha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_latte">
        <property name="text">
         <string>Caffe Latte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_decaf">
        <property name="text">
         <string>Decaf Caffe Americano</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_americano">
        <property name="text">
         <string>Caffe Americano</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cof_brewed">
        <property name="text">
         <string>Brewed Coffee</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="Do_Buttons">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>951</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_charge">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>160</y>
       <width>211</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Charge</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_print">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>210</y>
       <width>211</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Print Receipt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Amount</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="show_amount">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>80</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>₩</string>
     </property>
    </widget>
    <widget class="QListWidget" name="selected_items">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>681</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Show_imgs">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>301</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="QRcode_IMG">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="QR">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QR Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Receipt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Receipt IMG</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="Receipt_IMG">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>200</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
